application/features/bootstrap/SagePeopleIntegrationContext.php

```diff
@@ -14,22 +14,23 @@ public function __construct()
         echo 'Testing integration with Sage People.' . PHP_EOL;
         parent::__construct();
     }
-    
+
     /**
      * @Given I can make authenticated calls to the ID Store
      */
     public function iCanMakeAuthenticatedCallsToTheIdStore()
     {
         $this->idStore = new SagePeopleIdStore([
-            'authUrl'      => Env::requireEnv('TEST_SAGE_PEOPLE_CONFIG_authUrl'),
-            'queryUrl'     => Env::requireEnv('TEST_SAGE_PEOPLE_CONFIG_queryUrl'),
-            'clientId'     => Env::requireEnv('TEST_SAGE_PEOPLE_CONFIG_clientId'),
-            'clientSecret' => Env::requireEnv('TEST_SAGE_PEOPLE_CONFIG_clientSecret'),
-            'username'     => Env::requireEnv('TEST_SAGE_PEOPLE_CONFIG_username'),
-            'password'     => Env::requireEnv('TEST_SAGE_PEOPLE_CONFIG_password'),
+            'authUrl'         => Env::requireEnv('TEST_SAGE_PEOPLE_CONFIG_authUrl'),
+            'queryUrl'        => Env::requireEnv('TEST_SAGE_PEOPLE_CONFIG_queryUrl'),
+            'clientId'        => Env::requireEnv('TEST_SAGE_PEOPLE_CONFIG_clientId'),
+            'clientSecret'    => Env::requireEnv('TEST_SAGE_PEOPLE_CONFIG_clientSecret'),
+            'username'        => Env::requireEnv('TEST_SAGE_PEOPLE_CONFIG_username'),
+            'password'        => Env::requireEnv('TEST_SAGE_PEOPLE_CONFIG_password'),
+            'queryConditions' => Env::requireEnv('TEST_SAGE_PEOPLE_CONFIG_queryConditions'),
```

Comment:
Did you want this to truly be required? Since you provide a default, it seemed like maybe not.

Reply:
You're right, technically it is not required. But I intend to make it required at the next major version bump.